packages/ember-application/lib/system/application.js

```diff
@@ -693,9 +693,15 @@ var Application = Namespace.extend(DeferredMixin, {
     var namespace = this;
     var initializer;
 
+    var makeGraphValue = function(initializer) {
+      return function() {
+        return initializer.initialize.apply(initializer, arguments);
```

Comment:
@rwjblue It may may helpful when changing the arguments of initialize function in future (not sure).